docs/editors-guide/rare-disease-subset.md

```diff
@@ -0,0 +1,51 @@
+# Rare Disease Subset in Mondo
+
+A rare disease is a disease that affects fewer than 200,000 people in the United States. However, this definition is not universal, and it varies between rare disease sources, geographical locations, etc. Consequently, rare disease criteria also vary between sources, and one might have difficulties reconciling sources and creating a list for rare diseases. Mondo recognizes these impediments and differences of view. This document explains how Mondo handles rare diseases, how one could get a rare disease list, and the efforts made to bring the community together.
+
+#### What are rare disease in Mondo?
+We rely on rare disease authorities (see table below) to determine all diseases recognized as “rare”, according to their own criteria. When a disease is considered “rare” according to a source, the corresponding Mondo term is annoted with `in_subset` annotation expressing that this disease is considered rare according to this source: `X_rare`  (where X is the rare disease authority).
+
+Example: 'calciphylaxis' (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.
+
+![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)
+
+Some Mondo terms were annotated with a `‘bearer of’ some rare` axiom. These axioms were added automatically to terms coming from Orphanet (before the creation of 'orphanet_rare' in_subset annotation), and manually by a curator based on the disease definition.
+In our [2021 publication](https://mondo.monarchinitiative.org/pages/analysis/) ([and here](https://pubmed.ncbi.nlm.nih.gov/32020066/)), we identified rare diseases as
+- terms annotated with the `‘bearer of’ some rare` axiom (as provided by Mondo contributing resources)
+- **AND** limited to the ‘human disease’ branch (which excludes ‘susceptibility’ terms)
+_(note that the "gard subset" annotation reported in the publication has been ignored here)_
+
+The Mondo terms fulfilling the above conditions were annotated with the 'in_subset' annotation 'mondo_rare' (see arrows in image above).
+
+#### The Mondo rare disease subset include ALL rare diseases according to ALL rare disease sources.
+The Mondo rare disease subset represents the UNION of rare diseases according to all authority sources and Mondo. Most of the diseases in this subset will be viewed as “rare” by most authorities, but some will be viewed as “rare” by only some authorities.
```

Comment:
This is currently not true. There are currently 1/3 of the classes only supported by Mondo!